.travis.yml

```diff
@@ -29,8 +29,9 @@ before_script:
   - export TEST_DATABASE_URL=postgresql://postgres@localhost:5432/test
   - bash scripts/test_multiple_heads.sh
   
-script:
-  - nosetests tests/ -v --with-coverage
+script: |
+  nosetests tests/ -v --with-coverage -I tests/all/integration/api/helpers/test_auth.py
+  pytest tests/all/integration/api/helpers/test_auth.py
```

Comment:
Not cool

Reply:
I didn't know another way to do it. 
Somehow I have to exclude test_auth from nostests and test it with pytest